src/cluster_legacy.c

```diff
@@ -223,9 +231,11 @@ auxFieldHandler auxFieldHandlers[] = {
     {"nodename", auxHumanNodenameSetter, auxHumanNodenameGetter, auxHumanNodenamePresent},
     {"tcp-port", auxTcpPortSetter, auxTcpPortGetter, auxTcpPortPresent},
     {"tls-port", auxTlsPortSetter, auxTlsPortGetter, auxTlsPortPresent},
+    {"client-ipv4", auxClientIpV4Setter, auxClientIpV4Getter, auxClientIpV4Present},
+    {"client-ipv6", auxClientIpV6Setter, auxClientIpV6Getter, auxClientIpV6Present},
```

Comment:
nit - align with the other places?
```suggestion
    {"client-ipv4", auxAnnounceClientIpV4Setter, auxAnnounceClientIpV4Getter, auxAnnounceClientIpV4Present},
    {"client-ipv6", auxAnnounceClientIpV6Setter, auxAnnounceClientIpV6Getter, auxAnnounceClientIpV6Present},
```